src/Tests/Helpers/FundingPeriodAssertions.cs

```diff
@@ -19,9 +19,9 @@ public class FundingPeriodAssertions : GenericCollectionAssertions<FundingPeriod
         {
             Execute.Assertion
                 .Given(() => subjArray[i])
-                .ForCondition(v => v.AgreedPrice == targetValue)
+                .ForCondition(v => v.AgreedPrice*0.80m == targetValue)
                 .FailWith(
-                    $"Expected value {subjArray[i].AgreedPrice}] in Period[{i}] should be {targetValue}");
+                    $"Expected value {subjArray[i].AgreedPrice} in Period[{i+1}] should be {targetValue}");
```

Comment:
What is happening here?

Reply:
I agree with @tom-gough although I'd go further and say we shouldn't have logic in test code.

Reply:
This is no longer used.

Reply:
should we remove it then?

Reply:
Removed